openid-4-verifiable-presentations-1_0.md

```diff
@@ -3030,6 +3030,7 @@ The technology described in this specification was made available from contribut
    * remove x509_san_uri client identifier scheme
    * clarify that `dcql_query` and `presentation_definition` are passed as JSON objects (not strings) in request objects
    * support returning multiple presentations for a single dcql credential query when requested using `multiple`
+   * renamed "Client ID Scheme" to "Client ID Prefix"
```

Comment:
could we explicitly call out the renamed protocol elements here too please? (which I think is just the wallet metadata client id schemes supported?)

Reply:
Thanks, I updated the changelog